lab_1_keywords_tfidf/main.py

```diff
@@ -156,4 +258,15 @@ def extract_significant_words(chi_values: dict[str, float], alpha: float) -> Opt
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if chi_values and isinstance(chi_values, dict) and isinstance(alpha, float):
```

Comment:
revert the condition and return None immediately

Reply:
Done